web-ui/src/main/webapp/WEB-INF/jsp/registration.jsp

```diff
@@ -2,42 +2,75 @@
 <%@ taglib uri="http://www.springframework.org/tags/form" prefix="form"%>
 <html>
     <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
```

Comment:
You either reference frontend libraries (like Bootstrap) from an external source (from somewhere else on the web, like in this case here) or you host them yourself on your server. 

Choose a way.

If you use CDNs, you don't need to include Bootstrap in your project.